pages/index.js

```diff
@@ -1,7 +1,36 @@
+import pets from "@/pets";
+import styled from "styled-components";
+
+const StyledList = styled.ul`
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+  padding: 0;
+  align-items: center;
+  margin: 0;
+  height: 50vh;
+`;
+
+const StyledListItem = styled.li`
+  border: 2px black solid;
+  border-radius: 5px;
+  padding: 5px;
+  cursor: pointer;
```

Comment:
why is this indicating to be clickable, a listItem should never clickable. We only click on buttons or links, in general